components/x11/libXfont/mapfile-vers

```diff
@@ -183,13 +183,9 @@ SUNWprivate {
 	pcfWriteFont;
 	pcfError;
 
-	# "src/bitmap/snfstr.h" - needed by xfs
-	SnfSetFormat;
```

Comment:
https://lists.x.org/archives/xorg-devel/2014-July/043165.html - We likely don't want libxfont 1.5 unless we ship new Xorg